carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/utils/mobile/IMobileUtils.java

```diff
@@ -761,6 +763,20 @@ default public void zoom(int startx1, int starty1, int endx1, int endy1, int sta
             LOGGER.error("Error during zooming", e);
         }
     }
+
+    default boolean isAppRunning() {
+        String os = IDriverPool.getDefaultDevice().getOs();
```

Comment:
we have to avoid IDriverPool.getDefaultDriver() usage as much as possible.
That's temp method to support migration old static services to new model. Valid use-case to use:
getDevice().getOs();